### PR TITLE
react-intro draft

### PR DESCRIPTION
**Pull request by I3uckwheat**

### CODE REVIEW

File: frameworks/react-introduction.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+## Overview
+
+React is used most in conjuction with the react router to create single page web-application that feel like multi-page websites. This can make websites feel very fast when a user does not have to wait for different pages to load. It can also reduce strain on the server by having the client render the web page they request for you. 
+
+This framework's most defining feature is it's components. React was developed to be modular, and have easy to use 'parts' that can be plugged into different parts of your application. React heavly uses a syntax called JSX, which is an html like syntax for javascript. (This can be used without react, as it is just sytatic sugar over creating DOM elements manually using `document.createElement('div')`, for example). Anything written in react is simply javascript, and can be treated as such. React stays perfomrant by utilizing what they call a virtural DOM. The framework utilizes this virtural DOM by comparing it with the real DOM to tell which elements need to be updated and which ones do not. This keeps react fast.
```
Comment (RobPando) [inline frameworks/react-introduction.md]: heavily*...  syntactic* sugar... React stays performant*....

File: frameworks/react-introduction.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+This framework's most defining feature is it's components. React was developed to be modular, and have easy to use 'parts' that can be plugged into different parts of your application. React heavly uses a syntax called JSX, which is an html like syntax for javascript. (This can be used without react, as it is just sytatic sugar over creating DOM elements manually using `document.createElement('div')`, for example). Anything written in react is simply javascript, and can be treated as such. React stays perfomrant by utilizing what they call a virtural DOM. The framework utilizes this virtural DOM by comparing it with the real DOM to tell which elements need to be updated and which ones do not. This keeps react fast. 
+
+React is not only limited to the web browser, however. React Native can be used to develop cross-platofrm mobile applications with the same languages you are familliar with in the web browser.
```
Comment (RobPando) [inline frameworks/react-introduction.md]: Not sure "however" is necessary here? might be wrong tho.

File: frameworks/react-introduction.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+A note about Redux:
+
+When reading about the react framework, one might hear the term 'redux' thrown around. Redux is not exactly a part of react and can be used seperately from it. Redux is a way to manage the sate of an application. While react containers, (a type of react component), have state themselves: redux handles the state of the entire application by using a single point of truth, making the appliation easier to manage for a developer. If you don't know if you need redux, you probably don't, as stated on their website.
```
Comment (RobPando) [inline frameworks/react-introduction.md]: manage the state*

File: frameworks/react-introduction.md (hunk 1 above)
Comment (I3uckwheat) [reply frameworks/react-introduction.md]: Thank you! Spelling isn't my best attribute!

File: frameworks/react-introduction.md (hunk 2 above)
Comment (I3uckwheat) [reply frameworks/react-introduction.md]: You're right, i bet i was going into another thought

**Comment by I3uckwheat** [comment]
@RobPando @codyloyd @CouchofTomato Care to take a look?

**Pull request merged**